line/line.mjs

```diff
@@ -1,27 +1,85 @@
-import * as utils from '../utilities/shared.mjs'
+import * as utils from '../utilities/shared.mjs';
 
-export async function findLineById(id = null) {
-  let line = null
+export async function findLineById(id = null, options = {}) {
+  let line = null;
+
+  if (id === null || id === undefined) {
+    return line;
+  }
 
   if (!utils.validateID(id)) {
-    return line
+    return line;
   }
 
   const mockPause = new Promise((resolve) => {
     setTimeout(() => {
-      resolve(null)
-    }, 1500)
-  })
+      resolve(null);
+    }, 1500);
+  });
 
   const linesArray = [
     { id: 123, text: "Hey TPEN Works on 123" },
-  ]
+  ];
 
-  line = linesArray.find((line) => line.id === id) || null
+  line = linesArray.find((line) => line.id === id) || null;
 
   if (line === null) {
-    line = await mockPause
+    line = await mockPause;
+  }
+
+  if (line !== null) {
+    if (options.text === 'blob') {
+      return { text: line.textualBody };
```

Comment:
You will have to set the Content-Type for this.  This should return something that is "text" and not a JSON.  You would set the type to "text/plain" then return the text blob that is the text of ` line.textualBody`